minerl/Malmo/Minecraft/src/main/java/com/microsoft/Malmo/MissionHandlers/RewardForPossessingItemImplementation.java

```diff
@@ -267,5 +315,6 @@ public void getReward(MissionInit missionInit, MultidimensionalReward reward) {
     public void cleanup() {
         super.cleanup();
         MinecraftForge.EVENT_BUS.unregister(this);
+        MalmoMod.MalmoMessageHandler.deregisterForMessage(this, MalmoMessageType.SERVER_COLLECTITEM);
```

Comment:
Should you deregister SERVER_DISCARDITEM here as well?

Reply:
good point, thanks!

Reply:
done!